src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -162,14 +165,14 @@ class ActivityFeed extends React.Component<Props, State> {
     };
 
     render(): React.Node {
-        const { handlers, inputState, isLoading, permissions, translations } = this.props;
+        const { handlers, inputState, isLoading, permissions, translations, comments, tasks, versions } = this.props;
         const { approverSelectorContacts, mentionSelectorContacts, isInputOpen } = this.state;
         const { currentUser } = inputState;
         const showApprovalCommentForm = !!(currentUser && getProp(handlers, 'comments.create', false));
         const hasCommentPermission = getProp(permissions, 'comments', false);
         const hasTaskPermission = getProp(permissions, 'tasks', false);
 
-        const feedState = [];
+        const feedState = [].concat(comments, tasks, versions);
```

Comment:
[...comments, ...tasks, ...versions] works as well, but this will be need to be changed by Justin when he starts sorting.